man/git-ftp.1.md

```diff
@@ -559,7 +561,19 @@ hook. This hook is **not** bypassed by the --no-verify option.
 It is meant primarily for notification and its exit status does not have any
 effect.
 
-# NETRC
+# PASSWORDS
+
+If your password contains special characters you have to take it with care. In most cases it is a good idea to quote passwords with single quotes:
+
+	--passwd '#my$fancy!secret'
+
+Mostly `--ask-passwd` works even if `--passwd` does not work. So maybe you can give this a try.
+
+Quoting also works if setting the [default](#defaults) with `git config`:
+
+	$ git config git-ftp.password '#my$fancy!secret'
+
+## netrc
```

Comment:
Why did you change this to lower case?

Reply:
In case you mean the headline … Because it’s an H2 now and not an H1 anymore.

Reply:
But other H2s are upper case as well. For example: `## RESET THE UPLOADED STATE`

Reply:
Douh 😆